py/desispec/parallel.py

```diff
@@ -380,6 +381,13 @@ def stdouterr_redirected(to=None, comm=None, overwrite=False):
     # The DESI loggers.
     desi_loggers = desiutil.log._desiutil_log_root
 
+    def _rank_filename(base, rank):
```

Comment:
A very minor thing, but could this be called basename or something more descriptive? When I read base and rank I immediately expected two integers (though I didn't know what base referred to), but then base turned out to be a filename. It is obvious from the code what it is, but still.

Reply:
Good suggestion.  I updated it to "basename" and expanded the docstring.